notifications/subscribe-to-notification-channels.sh

```diff
@@ -0,0 +1,28 @@
+# >> START subscribe_to_notification_events Subscribe to notification events from the command line
```

Comment:
For the key `subscribe_to_notification_events`, using slugs instead of underscores is preferred but is solely style for consistency's sake. Anything that's not whitespace is fine for the key as long as the keys in the start/end markers match.